frontend/app/API.js

```diff
@@ -1,63 +1,6 @@
 import { browserHistory } from 'react-router'
 import Raven from 'raven-js'
 
-const parse = {
```

Comment:
@mhfowler I removed `parse` because we're not really doing any transformations on the data we receive from the backend, and using `parse` requires us to explicitly whitelist fields from our API payloads